### PR TITLE
Edits to EPRS: fix to offline snapshot steps PR5985

### PR DESCRIPTION
**Pull request by ebgitelman**

### CODE REVIEW

File: product_docs/docs/eprs/7/07_common_operations/09_offline_snapshot.mdx [hunk 1]
```diff
@@ -101,18 +101,21 @@ Perform these steps before creating any subscriptions:
...
     -   Set the `batchInitialSync` option to the appropriate setting for your situation as discussed at the end of [Non-batch mode synchronization](#non_batch_mode_sync).
 
-1.  If you modified the publication server configuration, reload the configuration. See [Reloading the Publication or Subscription Server Configuration File (reloadconf)](../05_smr_operation/02_creating_publication/01_registering_publication_server/#registering_publication_server) for directions on reloading the publication server's configuration.
+1.  If you modified the publication server configuration, reload the configuration. See [Reloading the publication or subscription server configuration file (reloadconf)](../08_xdb_cli/03_xdb_cli_commands/52_reload_conf_file/].
```
Comment (ebgitelman) [inline product_docs/docs/eprs/7/07_common_operations/09_offline_snapshot.mdx:104]:
```suggestion
1.  If you modified the publication server configuration, reload the configuration. See [Reloading the publication or subscription server configuration file (reloadconf)](../08_xdb_cli/03_xdb_cli_commands/52_reload_conf_file/).
```

File: product_docs/docs/eprs/7/07_common_operations/09_offline_snapshot.mdx [hunk 2]
```diff
@@ -141,21 +144,21 @@ Perform these steps before adding primary nodes:
     -   Set the `offlineSnapshot` option to `true`. When you restart the publication server or reload the publication server's configuration via reloadconf, this setting has the effect that adding a primary node sets a column in the control schema indicating an offline snapshot is used to load this primary node.
     -   Set the `batchInitialSync` option to the appropriate setting for your situation as discussed at the end of [Non-batch mode synchronization](#non_batch_mode_sync).
 
-1.  If you modified the publication server configuration file, reload the configuration. See [Reloading the Publication or Subscription Server Configuration File (reloadconf)](../05_smr_operation/02_creating_publication/01_registering_publication_server/#registering_publication_server) for directions to reload the publication server's configuration.
+1.  If you modified the publication server configuration file, reload the configuration. See [Reloading the publication or subscription server configuration file (reloadconf)](../08_xdb_cli/03_xdb_cli_commands/52_reload_conf_file/].
```
Comment (ebgitelman) [inline product_docs/docs/eprs/7/07_common_operations/09_offline_snapshot.mdx:147]:
```suggestion
1.  If you modified the publication server configuration file, reload the configuration. See [Reloading the publication or subscription server configuration file (reloadconf)](../08_xdb_cli/03_xdb_cli_commands/52_reload_conf_file/).
```

**Comment by ebgitelman** [comment]
@gvasquezvargas The step with all that explanation in it made the procedure hard to follow and didn't have the right information in the right places. See how you like this revision.

Some changes applied for the Adding a primary node procedure as well, so I made them in both places.

Also, the links in this topic are using a convention where instead of just linking to the document itself, it links to an HTML div in the document. That's fine for the subtopics that might not have built-in links. But for the full topics, you end up going to an random area under the document title, so you don't know if you've gone to the right place. Suggest changing these when you have a chance. I changed some, and there was also a link that was going to completely the wrong place altogether that I fixed.

**Review by gvasquezvargas — APPROVED** [review]
Thanks for the edit @ebgitelman ! LGTM!
I am going to figure out how I can best remove the div references in this file without breaking any existing changes, it shouldn't be too hard but I have to be smart about this, so I will do that in a separate PR.
